medical_his/models/medical_hospital_bed.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Tech-Receptives Solutions Pvt. Ltd.
+#    Copyright (C) 2004-TODAY Tech-Receptives(<http://www.techreceptives.com>)
+#    Special Credit and Thanks to Thymbra Latinoamericana S.A.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+from openerp.exceptions import ValidationError
+
+
+class MedicalHospitalBed(models.Model):
+    _name = 'medical.hospital.bed'
+    _inehrit = ['abstract.medical.hospital']
+    _description = 'Medical Hospital Bed'
+
+    def _get_selection_state(self):
```

Comment:
In 8.0 fields.Selection now has the attribute selection_add that you can use to extend choices. So you can simply define you selection lists in the field definition.